components/backup_text/src/bin/rewrite/br_models.rs

```diff
@@ -93,40 +95,54 @@ impl BrColumnInfo {
 #[derive(Debug, Default, Clone)]
 pub struct RewriteInfo {
     pub table_info: TableInfo,
-    pub table_name: String,
-    pub db_name: String,
+    pub name: String,
 }
```

Comment:
`TableInfo` already has the field `name`, `RewriteInfo` looks duplicated and can be replaced by `TableInfo`.